AdyenCard/Components/Card/CardComponent.swift

```diff
@@ -184,8 +175,7 @@ extension CardComponent: CardViewControllerDelegate {
     
     func didChangeBIN(_ value: String) {
         self.cardComponentDelegate?.didChangeBIN(String(value.prefix(publicBinLength)), component: self)
-        let parameters = CardBrandProviderParameters(bin: value, supportedTypes: supportedCardTypes)
-        cardBrandProvider.provide(for: parameters) { [weak self] binInfo in
+        binInfoProvider.provideInfo(for: value, supportedTypes: supportedCardTypes) { [weak self] binInfo in
```

Comment:
provideInfo() -> provide(), info is redundant, the info word is already in the class name

Reply:
👍